README.md

```diff
@@ -4,106 +4,66 @@ String interpolation of translated text and React components.
-const substitution = 10;
-
-tag`Some statement that can use ${substitution}`
-// => Some statement that can use 10
-```
-
-In this form we are not able to infer a sensible **token name**.
-
-Presuming we use the substitution index we would be left with a translation like this.
-
-```
-msgid "Some statement that can use __0__"
-msgstr ...
+```js
+gettext`foo`
 ```
 
-Obviously `__0__` is **not** a good placeholder. We need something more descriptive.
-
-We need to be equally explicit about the **token name** and **token value**.
-
-**(GOOD)**
+key/val:
 
-```javascript
-tag`Some statement that can use ${{num: substitution}}`
-// => Some statement that can use 10
+```js
+gettext`Welcome to ${{location: 'hell'}}`
 ```
 
-We provide an `object` with single key-value and achieve a translation like this.
+jsx (nested call):
 
+```js
+gettext`For more ${{link: <a href="http://google.com">{gettext`information`}</a>}}`
 ```
-msgid "Some statement that can use __num__"
-msgstr ...
-```
-
-While this may feel a little contrived we feel it is the most explicit solution.
 
-### with plain values
+### `ngettext`
 
-By default we require that all substitutions be object key-value as shown above.
+- simple value:
 
-Any substitution passed **by value** will be treated like a normal Template Literal and will **appear in your translation**.
-
-```
-const substitution = 'foo';
-tag`Some simple ${substitution}`
+```js
+ngettext(planets)`Hello world!|Hello worlds!`
 ```
+|   | input | output |
+|---|----|----|
+| **singular** | planets <= 1  | "Hello World!"  |
+| **plural**   | otherwise     | "Hello Worlds!" |
+<!-- Basic logic: If `planets > 1`  return "Hello world" otherwise -->
 
-Results in:
-
-```
-msgid "Some simple foo"
-msgstr ...
-```
-
-### with React
-
-Consider a React `Paginator` component where `More` and `Less` anchors will allow more or fewer results to be shown.
+- Object:
 
-```html
-<span>Show <a>10 More</a> or <a>5 Less</a></span>
+```js
+ngettext(daysLeft)`Only one day to go!|${{daysLeft}} days to go!`
 ```
 
-The anchors themselves would need to be separate HTML elements, nested in the overall statement. And the number of more or less items may vary. So it is reasonable to assume they would be nested React components.
+|   | input | output |
+|---|----|----|
+| **singular** | `{daysLeft: 1}`  | "Only one day to go!"  |
+| **plural**   | `{daysLeft: 9}`  | "9 days to go!" |
 
-In a perfect world we could use a **Tagged Template Literal** to compose idiomatically.
 
-```
-<span>
-  {tag`Show
-    ${{more: <More {...{numMore, onMore}} />} or
-    ${{less: <Less {...{numLess, onLess}} />}`}
-</span>
-```
-
-Presumably we will translate the `<More/>` and `<Less/>` components separately. But we will want our translation to still show some context.
-
-```
-msgid "Show __more__ or __less__"
-msgstr ...
-```
+For more details on usage and the projects motivation see [this documentation](docs/InDepth.md) or check out the examples directory.
```

Comment:
I would just dump this. I will go through later but I don't think there is anything to save. Maybe just point to the examples directory.

Reply:
I thought there was some stuff that was good if you want to go into more depth. For me, my motivation wan mostly "I want to get this working" which means I want to see examples and the briefest summary of how it works/what it does. 

I dont think theres anything wrong with more in depth information/explanation as long as its out of the way but I'll leave it up to you as to whether you want to dump the whole thing or not.

Reply:
Hal 👍 
It is good to have some "inDepth" thoughts, but separately from the main readme, heaps of useful info imo

Reply:
Yeah I thought so - the main thing for me is that you can "get into it" quickly if you have a fairly straightforward use case - as long as the more in depth isnt interfering in that process then I think its a good thing. But I'm happy to leave the final call up to Ben.